### DISCUSSION TITLE
Disable devices in bulk in advanced mode only

### DISCUSSION THREAD

**Discussion by jerome6994** [home-assistant/frontend#18055] [Configuration]
Good morning

I don't know if this idea has already been raised but on the device screen (the complete list of HA or each integration) is it possible to have a checkbox to activate or deactivate the device?
This would allow mass processing as on entities.
Possibility open in advanced mode only for example

I had the scenario on the Unifi Network integration where I had to make 520 times 5 clicks to deactivate each “unnamed device”
Afterwards I don't know why it comes back up! (I'm not a company I don't have all that on my network)!!

Thanks 

Jérôme

**Comment by Schrolli91** [comment]
Such a feature would be great.
Im using a FritzBox Router and his integration generates a lot of devices, where I only need some of them.

I can bulk deactivate the not needed entities. But it would be awesome if I can deactivate the whole devices to hide them everywhere.
